src/events/payloadsController.ts

```diff
@@ -2,23 +2,7 @@ import { IPayloadsControllerCore_ABI } from '@bgd-labs/aave-address-book';
 import { strategicGetLogs } from '@bgd-labs/js-utils';
 import { Address, Client, getAbiItem } from 'viem';
 
-export enum PayloadState {
-  None,
-  Created,
-  Queued,
-  Executed,
-  Cancelled,
-  Expired,
-}
-
-export const HUMAN_READABLE_PAYLOAD_STATE = {
```

Comment:
i saw it being removed but not added anywhere, do you already have sth similar to map from state to human readable word?

Reply:
no, do you need this?

Reply:
I need it yes, but i can move to my lib if you don't need it - just assumed, somewhere you must have it as here's you mapping from the enum value to a string somehow:
![image](https://github.com/bgd-labs/aave-governance-ui-helpers/assets/4396533/c19a19ad-a55e-48ee-8cab-dd7113248540)

Reply:
it's for proposal status

Reply:
I have in UI... but format not like you have...
I will return this